MREUnityRuntime/MREUnityRuntimeLib/App/MixedRealityExtensionApp.cs

```diff
@@ -381,88 +381,64 @@ internal bool OwnsActor(IActor actor)
 
         #region Methods - Private
 
-        private void ProcessCreateFromGLTFQueue()
+        [CommandHandler(typeof(CreateFromGLTF))]
+        private async Task OnCreateFromGLTF(CreateFromGLTF payload)
         {
-            // GLTFLoader can only load one model at a time :(
-            if (Monitor.TryEnter(_createFromGLTFQueue))
+            if (_actorManager.HasActor(payload.Actor?.Id) && _actorManager.IsActorReserved(payload.Actor?.Id))
```

Comment:
I think these conditions should be OR'd, not AND'd:
```ts
if (_actorManager.HasActor(payload.Actor?.Id) || _actorManager.IsActorReserved(payload.Actor?.Id))
```
(I realize this is my code from a while back :) )

Reply:
Is there a reason we succeeded in this case? My PR changes this to a failure, but I want to make sure that's the right behavior.

---
In reply to: [246835616](https://github.com/Microsoft/mixed-reality-extension-unity/pull/5#discussion_r246835616) [](ancestors = 246835616)

Reply:
Let's try going with failure and see how that goes.